src/Spout/Writer/XLSX/Internal/Worksheet.php

```diff
@@ -200,11 +200,22 @@ private function getCellXML($rowIndex, $cellNumber, $cellValue, $styleId)
         $cellXML .= ' s="' . $styleId . '"';
 
         if (CellHelper::isNonEmptyString($cellValue)) {
-            if ($this->shouldUseInlineStrings) {
-                $cellXML .= ' t="inlineStr"><is><t>' . $this->stringsEscaper->escape($cellValue) . '</t></is></c>';
+            $matches = array();
+            if (preg_match('/=HYPERLINK\("(.*)","(.*)"\)/', $cellValue, $matches)) {
```

Comment:
This regex should be changed to support other ways of writing the formula: `/=HYPERLINK\([\'"](.*)[\'"],\s*[\'"](.*)[\'"]\)/`. This support single and double quotes as well as any number of spaces after the comma (it may also need the same `\s` before the comma)